mz_strm_zstd.c

```diff
@@ -132,31 +129,21 @@ int32_t mz_stream_zstd_read(void *stream, void *buf, int32_t size) {
             zstd->in.src = zstd->buffer;
             zstd->in.size = read;
             zstd->in.pos = 0;
+            zstd->total_in += read;
         }
 
         total_out_before = zstd->out.pos;
         total_in_before = zstd->in.pos;
-
         result = ZSTD_decompressStream(zstd->zdstream, &zstd->out, &zstd->in);
-
         if (ZSTD_isError(result)) {
             zstd->error = (int32_t)result;
             return MZ_DATA_ERROR;
         }
-
-        total_in_after = zstd->in.pos;
```

Comment:
I would like to keep things the way they were with these extra variables. I realize that some of the variables are not necessary. But it keeps all the other streams the same way.